x-pack/plugins/security_solution/cypress/e2e/hosts/inspect.cy.ts

```diff
@@ -14,7 +14,8 @@ import { login, visit, visitHostDetailsPage } from '../../tasks/login';
 
 import { HOSTS_URL } from '../../urls/navigation';
 
-describe('Inspect', () => {
+// This will be fixed in a follow up PR
+describe.skip('Inspect', () => {
```

Comment:
Would you mind opening up an issue with these skipped tests and add it to the comments here?

Reply:
The issue is up https://github.com/elastic/kibana/issues/152359, and the PR https://github.com/elastic/kibana/pull/151714

Reply:
Thanks!